src/solver/conv_mlir_igemm_fwd_xdlops.cpp

```diff
@@ -139,14 +151,35 @@ bool PerformanceConvMlirIgemmXdlops::SetNextValue(const ConvolutionContext& /*co
             break;
         if(!NextTwoPower<16, 256>(GemmNPerBlock))
             break;
-        if(!NextTwoPower<1, 8>(GemmKPerBlock))
+        if(!NextTwoPower<4, 8>(GemmKPerBlock))
             break;
         if(!NextTwoPower<4, 128>(GemmMPerWave))
             break;
         if(!NextTwoPower<16, 128>(GemmNPerWave))
             break;
-        if(!NextTwoPower<4, 8>(GemmKPACKSize))
-            break;
+        if(config.IsFp32())
+        {
+            if(config.direction.IsForward())
+            {
+                if(!NextTwoPower<1, 4>(GemmKPACKSize))
+                    break;
+            }
+            else if(config.direction.IsBackwardWrW())
+            {
+                if(!NextTwoPower<1, 2>(GemmKPACKSize))
```

Comment:
What about 4/8?

Reply:
4 has known intermittent issue.
8 is not valid.

Reply:
Make sense.